app/services/affiliate-slots.js

```diff
@@ -86,13 +86,27 @@ export default Service.extend({
   currentCountry: readOnly('geo.country'),
 
   /**
-   * Get one unit that can be displayed on search page with given `query`
+   * Get one BIG unit that can be displayed on search page with given `query`
+   *
+   * @param {string} query
+   * @returns {AffiliateUnit|undefined}
+   */
+  getBigUnitOnSearch(query) {
```

Comment:
(feel free to ignore) but maybe we should call it 'Fan Suggested' or something else other than 'Big Unit'

Reply:
As this code is supposed (in the future) to handle different units I’d rather stick to big/small than to labels that the UI will have.